.travis.yml

```diff
@@ -13,4 +13,5 @@ install:
   - travis_retry composer install
 
 script:
-  - phpunit
\ No newline at end of file
+  - phpunit --exclude-group integration
+  - '[[ "${TRAVIS_PULL_REQUEST}" == "false" ]] && ( phpunit --group integration ) || ( echo "Testing PR - No integration tests available")'
```

Comment:
Why `"${TRAVIS_PULL_REQUEST}" == "false"`?

Is it to reduce the number of request to the server? If so, then it is okey (but weird). 

If it is to make sure we do not allow others for sending emails by making a PR? Then it's wrong. I'll just remove this line in by PR.